### ISSUE TITLE
Runtime SPIR-V VUIDs that could be validated standalone

### ISSUE THREAD

**Issue by Rua** [KhronosGroup/Vulkan-Docs#2267]
The Runtime SPIR-V section of the spec is supposed to contain things that can only be validated at runtime, because they rely on properties of the device, enabled features, or input data from the user. But the following VUIDs don't seem to rely on anything at runtime, they use only information that's known from the SPIR-V code. So they should probably be moved to the Standalone SPIR-V section.

- [VUID-RuntimeSpirv-DescriptorSet-06323](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-DescriptorSet-06323)
- [VUID-RuntimeSpirv-SubgroupUniformControlFlowKHR-06379](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-SubgroupUniformControlFlowKHR-06379) (the part about invocation repack instructions)
- [VUID-RuntimeSpirv-OpImageWeightedSampleQCOM-06980](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageWeightedSampleQCOM-06980)
- [VUID-RuntimeSpirv-OpImageWeightedSampleQCOM-06981](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageWeightedSampleQCOM-06981)
- [VUID-RuntimeSpirv-OpImageBlockMatchSSDQCOM-06983](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageBlockMatchSSDQCOM-06983)
- [VUID-RuntimeSpirv-OpImageBlockMatchSSDQCOM-06984](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageBlockMatchSSDQCOM-06984)
- [VUID-RuntimeSpirv-OpImageWrite-07112](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageWrite-07112)
- [VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08976](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08976)
- [VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08978](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08978)
- [VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08980](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08980)
- [VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08982](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpCooperativeMatrixMulAddKHR-08982)
- [VUID-RuntimeSpirv-scope-08984](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-scope-08984)
- [VUID-RuntimeSpirv-ShaderEnqueueAMDX-09191](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-ShaderEnqueueAMDX-09191)
- [VUID-RuntimeSpirv-NodePayloadAMDX-09192](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-NodePayloadAMDX-09192)
- [VUID-RuntimeSpirv-OpImageBlockMatchWindow-09220](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageBlockMatchWindow-09220)
- [VUID-RuntimeSpirv-OpImageBlockMatchWindow-09221](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/RuntimeSpirv.html#VUID-RuntimeSpirv-OpImageBlockMatchWindow-09221)

**Comment by spencer-lunarg** [comment]
>  VUID-RuntimeSpirv-OpImageWrite-07112

You need to know the descriptor bound to it in order to match the VkFormat

**Comment by Rua** [comment]
> You need to know the descriptor bound to it in order to match the VkFormat

If the format is specified in the shader itself, do you not know?

**Comment by gnl21** [comment]
> > You need to know the descriptor bound to it in order to match the VkFormat
>
> If the format is specified in the shader itself, do you not know how many components it has?

Yes, the table referred to in the VU gives the VkFormat corresponding to the format used in the SPIR-V code. It's not a reference to the actual format of the descriptor in this case. The VU that matches with the actual descriptor format it, for example, VUID-vkCmdDraw-OpImageWrite-08795.
